nulib/react-media-player
# Add fixtures dir, update WebVTT hook logic, add tracks to video.

## What does this do?

The pull request addresses a few issues that came up during the preparation for the Samvera Connect demonstration.

**In it we now have:**

-  a more formal directory of fixtures accessible to the browser when running `npm run dev`. This allows us to update and test locally hosted manifests and creates a structure for better on-the-fly swapping out of manifest varieties.
- updated logic in the hook that retrieves the `supplementing` WebVTT content resources to conform to updates to our manifests in https://github.com/nulib/repodev_planning_and_docs/issues/2320.
- addition of `<track>` elements within the `<video>` on the Player component. 

## How do we test this?

1. `npm run dev`
2. You'll see it fire up with a  manifest loaded that was generated by https://github.com/nulib/repodev_planning_and_docs/issues/2320.
3. The resources the media player attempts to retrieve will not be available, but you will see the requests in the console log. 
4. As long as nothing fatals we should be good to go.

![image](https://user-images.githubusercontent.com/7376450/137547206-f671e01a-c267-4d77-a9db-12ce905b7a13.png)
